src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient.swift

```diff
@@ -39,32 +39,34 @@ final class HTTPClient: Client {
 	) {
 		let url = configuration.availableHoursURL(day: day, country: country)
 
-		session.GET(url) { result in
-			switch result {
-			case let .success(response):
-				// We accept 404 responses since this can happen in case there
-				// have not been any new cases reported on that day.
-				// We don't report this as an error to simplify things for the consumer.
-				guard response.statusCode != 404 else {
-					completeWith(.success([]))
-					return
-				}
+		session.GET(url) { [weak self] result in
+			self?.queue.async {
+				switch result {
+				case let .success(response):
+					// We accept 404 responses since this can happen in case there
+					// have not been any new cases reported on that day.
+					// We don't report this as an error to simplify things for the consumer.
```

Comment:
This comment is not correct anymore, btw, but that's for a different PR I guess...